presctl/cmd/entities/templates.go

```diff
@@ -0,0 +1,142 @@
+package entities
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+const templateURLFormat = "http://%s:%s/api/v1/templates/%s/%s/%s"
+const actionURLFormat = "http://%s:%s/api/v1/projects/%s/%s"
+
+const (
+	create = 0
+	update = 1
+	delete = 2
+	get    = 3
+)
+
+type restOp int
+
+// if error is not null print it and exit the program
+func check(err error) {
```

Comment:
every error throws the application?

Reply:
yes. its a cli print the error and throw